testing/trino-testing-services/src/main/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

```diff
@@ -0,0 +1 @@
+io.trino.junit.ReportBadJunitTestAnnotations
```

Comment:
This is very brittle. It only works if a module depends on trino-testing-services. To ensure it applies to everything, the extension should be registered via the surefire plugin configuration. It may require moving this check out of the project, and building an artifact that the Trino build can depend on.

Reply:
I know, but it's much better than no reporter.